### PR TITLE
fix typo in filename (buider->builder); conform import to pep8

### PR DESCRIPTION
**Pull request by joseims**
Tests are missing

### CODE REVIEW

**Comment by joseims** [comment]
Everything that the k8s-grafana plugin uses is either not fixed or validated elsewhere

File: visualizer/plugins/k8s_grafana/plugin.py [hunk 1]
```diff
@@ -16,9 +16,7 @@
 # limitations under the License.
 
 import time
-
```
Comment (clenimar) [inline visualizer/plugins/k8s_grafana/plugin.py]: please keep these blank lines.

**Review by clenimar — CHANGES_REQUESTED** [review]
is this still a WIP? title, description and the set of modifications are confusing.

**Comment by joseims** [comment]
After analyzing the code i found out that no validation is necessary since all the variables used by the k8s-grafana are either not fixed or validated elsewhere. So i just reused the pull request to rename the file visualizer/tests/unit/plugins/test_buider.py to  visualizer/tests/unit/plugins/test_builder.py.

Related to issue #20

**Review by matheuscmelo — CHANGES_REQUESTED** [review]
Could you please squash these commits? Seems to be a lot of them for just too little modifications. Besides, LGTM.

**Comment by clenimar** [comment]
agreed. you've got way more commits than needed. try to keep things simple. one change per commit, and a (really small) highly-related set of commits per PR. also improve the title and description to reflect better what you're proposing.

suggested rewording: 
fix typo in filename (buider->builder); conform import to pep8

cheers.

**Review by matheuscmelo — APPROVED** [review]

**Review by clenimar — APPROVED** [review]

**Pull request merged**